examples/source_examples/aws_athena/using_boto3/connector.py

```diff
@@ -0,0 +1,125 @@
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+import boto3
+import json  # Import the json module to handle JSON data.
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+import time
+
+TABLE_NAME = "test_rows"
+
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function:
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+def schema(configuration: dict):
+    return [
+        {
+            "table": "customers",  # Name of the table in the destination.
+            "primary_key": ["customer_id"],  # Primary key column(s) for the table.
+            "columns": {  # Define the columns and their data types.
+                "customer_id": "STRING",  # Integer column for the customer_id.
+                "first_name": "STRING",  # String column for the first name.
+                "last_name": "STRING",  # String column for the last name.
+                "email": "STRING",  # String column for the email.
+            },
+        }
+    ]
+
+def create_athena_client(configuration):
+    return boto3.client('athena',
+                        aws_access_key_id=configuration["aws_access_key_id"],
+                        aws_secret_access_key=configuration["aws_secret_access_key"],
+                        region_name=configuration['region_name'])
+
+
+def get_query_results(athena_client, query_execution_id):
+
+    response = athena_client.get_query_results(QueryExecutionId=query_execution_id)
+
+    while True:
+        response = athena_client.get_query_results(QueryExecutionId=query_execution_id,
+                                                   NextToken=response['NextToken']) \
+            if 'NextToken' in response else athena_client.get_query_results(QueryExecutionId=query_execution_id)
+
+        # Skipping first row as it contains only metadata
+        for row in response['ResultSet']['Rows'][1:]:
+            row_data = []
+            for cell in row['Data']:
+                if 'VarCharValue' in cell:
+                    row_data.append(cell['VarCharValue'])
+                elif 'BigIntValue' in cell:
+                    row_data.append(int(cell['BigIntValue']))
+                elif 'DoubleValue' in cell:
+                    row_data.append(float(cell['DoubleValue']))
+                elif 'BooleanValue' in cell:
+                    row_data.append(bool(cell['BooleanValue']))
+                else:
+                    row_data.append(None)  # Handle unexpected data types
```

Comment:
it should throw some exception?

Reply:
Removed the comment. It handles null values.